google/cloud/security/scanner/scanner.py

```diff
@@ -200,42 +200,65 @@ def _flatten_violations(violations, flattening_scheme):
     """
 
     # TODO: Make this nicer
-    LOGGER.info('Writing violations to csv...')
```

Comment:
Can you re-write this TODO to be "Write customer flattening methods for each violation type."